src/operator/nn/mkldnn/mkldnn_batch_norm-inl.h

```diff
@@ -185,21 +119,20 @@ class MKLDNNBNForward {
 template<typename DType>
 static MKLDNNBNForward &GetBNForward(const BatchNormParam& param,
                                      const OpContext &ctx, const mkldnn::memory *data_mem,
-                                     unsigned flags) {
+                                     mkldnn::normalization_flags flags) {
 #if DMLC_CXX11_THREAD_LOCAL
   static thread_local std::unordered_map<MKLDNNBNSignature, MKLDNNBNForward, OpHash> fwds;
 #else
   static MX_THREAD_LOCAL std::unordered_map<MKLDNNBNSignature, MKLDNNBNForward, OpHash> fwds;
 #endif
   MKLDNNBNSignature key(param);
   key.AddSign(ctx.is_train);
-  key.AddSign(param.use_global_stats);
```

Comment:
use_global_stats should be added to key, as BN needs to check both is_train and use_global_stats

Reply:
MKLDNNBNSignature key(param) will hash param structure which include param.use_global_stats

Reply:
ok